balena.yml

```diff
@@ -15,13 +15,17 @@ assets:
         https://raw.githubusercontent.com/ipv6-router/master/ipv6-logo.png
 data:
   applicationEnvironmentVariables:
-    - CLIENTS_WHITELIST: 2001:db8:1::
+    - CLIENTS_WHITELIST: '2001:db8:1::abba;2001:db8:1::cafe'
```

Comment:
Before this PR, `balena push` was failing with:
```
[Error]    The included balena.yml is not a valid contract, allowed types are sw.application,sw.block,sw.hal-module, stopping build
```
And I gather it was because `2001:db8:1::` was unquoted and the last colon interpreted by Yaml as an object definition. Quoting the value avoids the error.